test/unittests/mongoBackend/mongoSubscribeContext_test.cpp

```diff
@@ -797,7 +797,7 @@ TEST(mongoSubscribeContext, Ent1_AttrN_CNbis)
     EXPECT_FALSE(sub.hasField("lastNotification"));
     EXPECT_FALSE(sub.hasField("throttling"));
     EXPECT_STREQ("http://notify.me", C_STR_FIELD(sub, "reference"));
-    EXPECT_STREQ("JSON", C_STR_FIELD(sub, "format"));
+    EXPECT_STREQ("legacy", C_STR_FIELD(sub, "format"));
```

Comment:
Note that "legacy" is the word used in the API, but the keyword used at DB level doesn't change (due to backward compability) and is remains as "JSON".